test_utilities/src/aws.rs

```diff
@@ -23,7 +23,7 @@ use crate::utilities::{build_platform_local_docker, FuncTestsSecrets};
 pub const AWS_REGION_FOR_S3: &str = "eu-west-3";
 pub const AWS_TEST_REGION: &str = "eu-west-3";
 pub const AWS_KUBERNETES_MAJOR_VERSION: u8 = 1;
-pub const AWS_KUBERNETES_MINOR_VERSION: u8 = 19;
+pub const AWS_KUBERNETES_MINOR_VERSION: u8 = 20;
```

Comment:
maybe move this update into another PR?

Reply:
did this to avoid waiting for first pipeline due to version change + merge before doing this grafana change.